doc/api/synopsis.md

```diff
@@ -9,9 +9,43 @@ Please see the [Command Line Options][] document for information about
 different options and ways to run scripts with Node.js.
 
 ## Example
-
 An example of a [web server][] written with Node.js which responds with
-`'Hello World'`:
+`'Hello World!'`:
+
+Firstly, Make sure you have downloaded Node.js from [Node.js Official website](http://nodejs.org/#download).
+Then, Follow this [installation guide](https://nodejs.org/en/download/package-manager/).
+
+
+Now, Create an empty project folder called `projects`, navigate into it:
+
+Linux and Mac:
+
```

Comment:
@gibfahn Note my previous comments about personal pronouns. I wonder if this should really be a guide so that we don't have to worry about that sort of thing. Link to it from the docs?